docs/deprecated-blocks.md

```diff
@@ -0,0 +1,86 @@
+# Deprecated Blocks
+
+When updating static blocks markup and attributes, block authors need to consider existing posts using the old versions of their block. In order to provide a good upgrade path, you can choose one of the following strategies:
+
+ - Do not deprecate the block and create a new one (a different name)
+ - Provide a "deprecated" version of the block allowing users opening these blocks in Gutenberg to edit them using the updated block.
+
+A block can have several deprecated versions, Gutenberg will try them one after another until finding the one that matches the saved markup.
```

Comment:
Grammar: Probably more appropriate to use a period (".") in place of the comma here.